supervisord/utils.conf

```diff
@@ -41,3 +41,12 @@ command=%(ENV_HOME)s/wait-for-it.sh %(ENV_CVAT_REDIS_HOST)s:6379 -t 0 -- bash -i
     "
 environment=VECTOR_EVENT_HANDLER="SynchronousLogstashHandler"
 numprocs=1
+
+[program:rqworker_cleaning]
```

Comment:
I think you also need to update the Helm chart to run the new worker.

Reply:
We have also `rqworker_notifications` in utils and I don't see it in helm (maybe I missed something).
@azhavoro, Could you please help me resolve comments related to helm?

Reply:
I think we should to be consistent and not add a new worker pod to the helm if the worker is running inside the utils container when deploying with docker-compose.

Reply:
Okay, my bad. I thought this was a new supervisord config.